src/js/libcs/OpImageDrawing.js

```diff
@@ -385,3 +385,19 @@ evaluator.cameravideo$0 = function() {
     });
     return img;
 };
+
+evaluator.playvideo$1 = function(args, modifs) {
+    var img = imageFromValue(evaluateAndVal(args[0]));
+    if (img.live) {
+        img.img.play();
+    }
+    return nada;
+};
+
+evaluator.pausevideo$1 = function(args, modifs) {
+    var img = imageFromValue(evaluateAndVal(args[0]));
+    if (img.live) {
+        img.img.pause();
+    }
+    return nada;
+};
```

Comment:
I'd like to see some duck type check on whether `img.img.play` and `img.img.pause` actually exist before calling them. For example, a texture containing the result of processing some WebGL video would be live as well, in the sense that it should not be cached but must be redrawn. Nevertheless, such a texture doesn't have `play` or `pause`. So simply do `if (img.live && img.img.play)`.